_ml-commons-plugin/cluster-settings.md

```diff
@@ -246,3 +276,23 @@ plugins.ml_commons.model_auto_redeploy_success_ratio: 0.8
 
 - Default value: 0.8
 - Value range: [0, 1]
+
+## Run Python-based models
+
+When set to `true`, this setting enables the ability to run Python-based models, such as [Metrics correlation]({{site.url}}{{site.baseurl}}/ml-commons-plugin/algorithms/#metrics-correlation).
```

Comment:
```suggestion
When set to `true`, this setting enables the ability to run Python-based models supported by OpenSearch, such as [Metrics correlation]({{site.url}}{{site.baseurl}}/ml-commons-plugin/algorithms/#metrics-correlation).
```

Reply:
I'm not sure if supported will be the right word here. 

For example: these [python based pre-trained models](https://opensearch.org/docs/latest/ml-commons-plugin/pretrained-models/), we didn't develop those, we just traced it and we are supporting this.

But `enable_inhouse_python_model` is only applicable if any model is developed in python by opensearch team. I hope this clarifies the distinction here.